trino-aws-proxy/src/main/java/io/trino/aws/proxy/server/security/opa/OpaS3SecurityConfig.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.aws.proxy.server.security.opa;
+
+import io.airlift.configuration.Config;
+import jakarta.validation.constraints.NotNull;
+
+import java.net.URI;
+
+import static java.util.Objects.requireNonNull;
+
+public class OpaS3SecurityConfig
+{
+    private URI opaServerBaseUri;
+    private String dataPath = "/v1/data/policy";
+
+    @NotNull
+    public URI getOpaServerBaseUri()
+    {
+        return opaServerBaseUri;
+    }
+
+    @Config("opa-s3-security.server-base-uri")
+    public OpaS3SecurityConfig setOpaServerBaseUri(String opaServerBaseUri)
+    {
+        this.opaServerBaseUri = URI.create(opaServerBaseUri);
+        return this;
+    }
+
+    @NotNull
+    public String getDataPath()
+    {
+        return dataPath;
+    }
+
+    @Config("opa-s3-security.data-path")
```

Comment:
Could we call this something like `policy-path`?

Reply:
This always gets appended to `opaServerBaseUri` - is there any benefit to this or could it be merged into the base URI?

If the security mapper wishes to change the URI, it can do so in the `serverUri` method

Reply:
I got rid of it